### PR TITLE
adding faction template files

### PR DESCRIPTION
**Pull request by Poornima545**

### CODE REVIEW

File: faction/index.html [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="faction-header">
+        <h1>faction</h1>
+        <input type="checkbox" id="menu-bar" class="menu-bar">
+        <label for="menu-bar" class="menu-icon">
+            <span></span>
```
Comment (naguvasanad) [inline faction/index.html]: qn: why are these empty span elements

File: faction/index.html [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+            <header class="feature-header">
+                <h1 class="heading">Sed mattis nunc lorem</h1>
+                <p class="content">Phasellus sed ultricies mi non congue ullam corper. Praesent tincidunt sed tellus ut
+                    rutrum. Sed
+                    vitae justo condimentum, porta lectus vitae, ultricies congue gravida diam non fringilla lorem.</p>
+            </header>
+
+            <article class="feature-list">
+                <i class="bi bi-gem"></i>
+                <div>
```
Comment (naguvasanad) [inline faction/index.html:42]: sug: instead of div sections can be used within article or vice-versa

File: faction/index.html [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+                    <h3 class="text">Nisl sed vitae</h3>
+                    <p class="content">Nullam et orci sed lorem magna et vivamus sagittis libero.</p>
+                </div>
+            </article>
+
+            <article class="feature-list">
+                <i class="bi bi-folder-symlink"></i>
+                <div>
+                    <h3 class="text">Sed mattis nunc</h3>
```
Comment (naguvasanad) [inline faction/index.html:52]: sug: can use better class instead of text

File: faction/index.html [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+                <div>
+                    <h3 class="text">Porta ultricies</h3>
+                    <p class="content">Maecenas sagittis felis ac sagittis semper. Curabitur purus leo.</p>
+                </div>
+            </article>
+
+            <article class="feature-list">
+                <i class="bi bi-send-fill"></i>
+                <div>
+                    <h3 class="text">Magna tellus</h3>
```
Comment (naguvasanad) [inline faction/index.html:70]: sug: same as above comment

File: faction/index.html [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+                    <h3 class="text">Sed mattis nunc</h3>
+                    <p class="content">Duis non efficitur nisi, sed justo. Maecenas sagittis semper.</p>
+                </div>
+            </article>
+
+            <article class="feature-list">
+                <i class="bi bi-pc-display-horizontal"></i>
+                <div>
+                    <h3 class="text">Porta ultricies</h3>
+                    <p class="content">Maecenas sagittis felis ac sagittis semper. Curabitur purus leo.</p>
```
Comment (naguvasanad) [inline faction/index.html:62]: sug: better class names can be used instead of content

File: faction/index.html [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+                    <h3 class="text">Magna tellus</h3>
+                    <P class="content">Auctor orci eu dolor consectetur, interdum ullamcorper ante.</P>
+                </div>
+            </article>
+        </section>
+
+        <hr>
+        <section class="gallery">
+            <article class="gallery-item">
+                <img src="./assets/pic01.jpg" alt="">
```
Comment (naguvasanad) [inline faction/index.html]: sug: missing value for an alt attribute

File: faction/index.html [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+                <h3 class="text">Sit amet posuere risus amet lorem faucibus lectus</h2>
+                    <p class="content">Fusce tincidunt, nisl eget mattis egestas, purus ipsum consequat orci, sit amet lobortis lorem
+                        lacus in tellus. Sed ac magna consequat.</p>
+                    <a href="#">Learn more</a>
+            </article>
+
+            <article class="gallery-item">
+                <img src="./assets/pic02.jpg" alt="">
```
Comment (naguvasanad) [inline faction/index.html]: sug: same as above comment

File: faction/index.html [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+                <h3 class="text">Ante sed fermentum tempus vitae eleifend magna</h2>
+                    <p class="content">Quam vel tortor tincidunt suscipit. Nullam auctor orci eu dolor consectetur, interdum ullamcorper
+                        ante tincidunt. Mauris felis felis elementum.</p>
+                    <a href="#">Learn more</a>
+            </article>
+
+            <article class="gallery-item">
+                <img src="./assets/pic03.jpg" alt="">
```
Comment (naguvasanad) [inline faction/index.html]: sug: same as above comment

File: faction/style.css [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+.feature-list {
+    border-top: 1px solid;
+    border-color: rgba(144, 144, 144, 0.25);
+    padding-top: 20px;
+    display: flex;
+    gap: 20px;
+}
+
+.text {
+    color: #3ed8c4;
```
Comment (naguvasanad) [inline faction/style.css:92]: sug: use unique color codes

File: faction/style.css [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+    margin: 0 0 10px 0;
+    font-size: 19px;
+    font-weight: 300;
+}
+
+.feature-list .bi {
```
Comment (naguvasanad) [inline faction/style.css:98]: sug: should not use default class names as selectors

File: faction/index.html [hunk 11]
```diff
@@ -0,0 +1,221 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Faction</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="faction-header">
```
Comment (preethamntp) [inline faction/index.html]: class name could have been better

File: faction/index.html (hunk 1 above)
Comment (preethamntp) [reply faction/index.html]: @Poornima545 can it be done without span ? you also can try using svg / some unicode in css

File: faction/index.html [hunk 12]
```diff
@@ -0,0 +1,221 @@
...
+                </div>
+            </article>
+        </section>
+
+        <hr>
+        <section class="gallery">
+            <article class="gallery-item">
+                <img src="./assets/pic01.jpg" alt="">
+                <h3 class="text">Sit amet posuere risus amet lorem faucibus lectus</h2>
+                    <p class="content">Fusce tincidunt, nisl eget mattis egestas, purus ipsum consequat orci, sit amet lobortis lorem
```
Comment (preethamntp) [inline faction/index.html]: space between the elements are not matching

File: faction/index.html [hunk 13]
```diff
@@ -0,0 +1,221 @@
...
+                        lacus in tellus. Sed ac magna consequat.</p>
+                    <a href="#">Learn more</a>
```
Comment (preethamntp) [inline faction/index.html]: no styles added

**Pull request merged**